src/Printer/FailuresPrinter.php

```diff
@@ -45,6 +47,11 @@ public function onEngineEnd(): void
             $this->printFailuresHeading($outcome, $style);
 
             if ($outcome === TestIssue::Deprecation) {
+                $display = filter_var($this->config->getRootAttribute('displayDetailsOnTestsThatTriggerDeprecations'), FILTER_VALIDATE_BOOLEAN);
+                if (! $display) {
```

Comment:
Can we please wrap this in a private method? This will help in further evolutions, like #197

```suggestion
                if ($this->shouldSkipPrinting($outcome)) {
```